velox/common/caching/tests/AsyncDataCacheTest.cpp

```diff
@@ -30,8 +30,8 @@ class AsyncDataCacheTest : public testing::Test {
  protected:
   static constexpr int32_t kNumFiles = 100;
   void initializeCache(int64_t maxBytes) {
-    cache_ =
-        std::make_shared<AsyncDataCache>(MappedMemory::getInstance(), maxBytes);
+    cache_ = std::make_shared<AsyncDataCache>(
+        MappedMemory::createDefaultInstance(), maxBytes);
```

Comment:
MappedMemory::createDefaultInstance() - it would be nice to stop using the default instance